modules/xcode/_preload.lua

```diff
@@ -6,6 +6,25 @@
 
 	local p = premake
 
+--
+-- Register the ios extension
+--
+
+	local osoption = p.option.get("os")
```

Comment:
This should be part of the `os` option by default, I've used `gmake`, `gmake2` and `xcode4` for iOS builds. Same thing goes for the `os.systemTags`.

Reply:
I see. I will do so.